schema/import/import_types.isl

```diff
@@ -0,0 +1,35 @@
+schema_header::{
+  imports: [
+    { id: "schema/import/abcde.isl", type: a, as: a2 },
+    { id: "schema/util/positive_int.isl" },
+    { id: "schema/import/abcde.isl", type: b },
+    { id: "schema/util/positive_int.isl", type: positive_int, as: posint },
+    { id: "schema/import/abcde.isl", type: c, as: c2 },
```

Comment:
Not blocking but use cases that I think we should have tests on 

1. Can I import the isl I am currently defining? So in this file can I have 
```
{ id : "schema/import/import_types.isl" } 
```
1. A  test to see if there is any weird (unwanted) shadowing going on e.g., 
```
{ id: "schema/import/abcde.isl", type: a, as: b},
{ id: "schema/import/abcde.isl", type: b, as: a}
```

Reply:
Spawned new issues (#4 and #5) to capture these test ideas.